contracts/Org.sol

```diff
@@ -18,23 +18,27 @@ import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
  */
 contract Org is Administratable {
   using SafeERC20 for IERC20;
-  // ========== STATE VARIABLES ==========
+
+  // ========== STRUCTS & EVENTS ==========
 
   struct Claim {
     string firstName;
     string lastName;
     string eMail;
     address desiredWallet;
-    bool filesSubmitted;
   }
 
+  event CashOutComplete(uint256 cashOutAmount);
+  event ClaimCreated(string claimId, Claim claim);
+  event ClaimApproved(string claimId, Claim claim);
+  event ClaimRejected(string claimId, Claim claim);
+
+  // ========== STATE VARIABLES ==========
+
   uint256 public taxId;
```

Comment:
If we have active claim this is not needed right?

Reply:
The taxId is for the Organization, and is set when it's created/deployed. The claim does not have an EIN.

Reply:
Oh duh yup